src/pages/AddPage/AddPage.jsx

```diff
@@ -1,12 +1,15 @@
-import React from "react";
+import React, { useContext } from "react";
 import "./AddPage.scss";
 import { useState } from "react";
 import { apiBaseUrl, requestDefaultHeaders } from "../../../config";
+import { StatusLengthContext } from "../../../StatusLengthContext";
 
 const AddPage = () => {
   const [text, setText] = useState("");
   const [error, setError] = useState(null);
   const [isSuccess, setIsSuccess] = useState(false);
+  const { allLength, setAllLength, uncompletedLength, setUncompletedLength } =
+    useContext(StatusLengthContext);
 
   const postTodo = async () => {
     try {
```

Comment:
Before trying post. Clean up the error and success state.

Reply:
Done.